### DISCUSSION TITLE
Filtering promotions based on price lists

### DISCUSSION THREAD

**Discussion by matteovava** [medusajs/medusa#10433] [Q&A]
Hi everyone,

I’m working on a scenario where I want to filter a promotion for a specific product based on the price list applied to that product for a customer.

Here’s my setup:
	•	I have a customer assigned to a customer list.
	•	A price list is associated with this customer list, overriding the standard product prices.

What I’d like to do is avoid applying a discount to products when they’re priced based on this specific price list. Is this behavior supported? If not, what would be the best approach to achieve this?

Thanks in advance for your help!

**Comment by srindom** [comment]
If I am understanding correctly the price in the price list is already reduced so you don’t want to allow additional reductions to prices when the price is fetched from the price list?

**Reply by matteovava** [reply]
> If I am understanding correctly the price in the price list is already reduced so you don’t want to allow additional reductions to prices when the price is fetched from the price list?

Yes, exactly this scenario
